new triggered camera is not installed

It's causing downstream packages to fail: 
Like this: http://build.ros.org/view/Kbin_uxv8_uXv8/job/Kbin_uxv8_uXv8__pr2_gazebo_plugins__ubuntu_xenial_arm64__binary/17/console
```
13:54:25 CMake Error at /opt/ros/kinetic/share/gazebo_plugins/cmake/gazebo_pluginsConfig.cmake:148 (message):
13:54:25   Project 'pr2_gazebo_plugins' tried to find library
13:54:25   'gazebo_ros_triggered_camera'.  The library is neither a target nor
13:54:25   built/installed properly.  Did you compile project 'gazebo_plugins'? Did
13:54:25   you find_package() it before the subdirectory containing its code is
13:54:25   included?
13:54:25 Call Stack (most recent call first):
13:54:25   /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:76 (find_package)
13:54:25   CMakeLists.txt:24 (find_package)
13:54:25 
```

--- a/gazebo_plugins/CMakeLists.txt
+++ b/gazebo_plugins/CMakeLists.txt
@@ -304,8 +304,10 @@ install(TARGETS
   gazebo_ros_utils
   gazebo_ros_camera_utils
   gazebo_ros_camera
+  gazebo_ros_triggered_camera
   gazebo_ros_multicamera
   MultiCameraPlugin
+  gazebo_ros_triggered_multicamera
   gazebo_ros_depth_camera
   gazebo_ros_openni_kinect
   gazebo_ros_openni_kinect

